esp/src/stub.htm

```diff
@@ -34,6 +34,15 @@
             document.write("<script type=\"text/javascript\" src=\"/esp/files/dist/node_modules.js\" charset=\"utf-8\"><" + "/script>");
             document.write("<script type=\"text/javascript\" src=\"/esp/files/dist/stub.eclwatch.js\" charset=\"utf-8\"><" + "/script>");
         }
+
+        checkCookie ();
+        function checkCookie(){
+            if (document.cookie.indexOf ("ESPAuthenticated") > -1){
+                window.location;
```

Comment:
Isn't `window.location.reload();` what your looking for?

Reply:
No, when using window.location.reload(); the checkCookie process gets stuck in an endless loop that never lets stub load. I will change this to window.location();